requirements.txt

```diff
@@ -2,7 +2,7 @@ setuptools
 typing_extensions
 
 numpy>=1.9.0
-scipy>=1.7.0
+scipy>=0.14.1
 
```

Comment:
This will need to be changed back, `SMAC==1.0.0` which is an underlying dependancy of `autosklearn=0.13.0` requires `scipy>=1.7`